frontend/packages/operator-lifecycle-manager/src/style.scss

```diff
@@ -313,3 +313,21 @@ $co-affinity-row-margin: 15px;
   display: flex;
   align-items: center;
 }
+
+.co-operator-uninstall-modal__operands-table {
+  max-height: 280px;
+  overflow-y: auto;
+  overflow-x: hidden;
+  margin-bottom: 16px;
+
+  .pf-c-toolbar {
+    padding: 0;
+  }
+  .co-m-nav-title {
+    margin: 0;
+  }
+  .co-m-pane__body {
+    padding-left: 0px;
+    padding-right: 0px;
+  }
```

Comment:
Don't override CSS this way. It's too hard to maintain and changes to the table CSS classes or styles could break this dialog. Much better is to use a specific class names for these elements in the operand table and set the styles you want on those classes (if it's possible a class names through). But I question if we should use the full resource table component in this context. It's not really meant for a dialog.